src/Language/Dutch.php

```diff
@@ -0,0 +1,423 @@
+        "Of",
+        "T",
+        "Unter",
+        "voor",
+        "am",
+        "d’",
+        "die le",
+        "of",
+        "t",
+        "unter",
+        "Voor ’t",
+        "Am de",
+        "Da",
+        "Do",
+        "Onder",
+        "Te",
+        "Van",
+        "voor ’t",
+        "am de",
+        "da",
+        "do",
+        "onder",
+        "te",
+        "van",
+        "Voor de",
+        "Auf",
+        "Dal",
+        "Don",
+        "Onder ’t",
+        "Ten",
+        "Van ’t",
+        "voor de",
+        "auf",
+        "dal",
+        "don",
+        "onder ’t",
+        "ten",
+        "van ’t",
+        "Voor den",
+        "Auf dem",
+        "Dal’",
+        "Dos",
+        "Onder de",
+        "Ter",
+        "Van de",
+        "voor den",
+        "auf dem",
+        "dal’",
+        "dos",
+        "onder de",
+        "ter",
+        "van De",
+        "Voor in ’t",
+        "Auf den",
+        "Dalla",
+        "Du",
+        "Onder den",
+        "Tho",
+        "van de",
+        "voor in ’t",
+        "auf den",
+        "dalla",
+        "du",
+        "onder den",
+        "tho",
+        "Van de l",
+        "Voor in t",
+        "Auf der",
+        "Das",
+        "El",
+        "Onder het",
+        "Thoe",
+        "van de l",
+        "voor in t",
+        "auf der",
+        "das",
+        "el",
+        "onder het",
+        "thoe",
+        "Van de l’",
+        "Vor",
+        "Auf ter",
+        "De",
+        "Het",
+        "Onder t",
+        "Thor",
+        "van de l’",
+        "vor",
+        "auf ter",
+        "de",
+        "het",
+        "onder t",
+        "thor",
+        "Van Den",
+        "Vor der",
+        "Aus",
+        "De die",
+        "I",
+        "Op",
+        "To",
+        "Van den",
+        "vor der",
+        "aus",
+        "de die",
+        "i",
+        "op",
+        "to",
+        "van den",
+        "Zu",
+        "Aus ’m",
+        "De die le",
+        "Im",
+        "Op ’t",
+        "Toe",
+        "Van Der",
+        "zu",
+        "aus ’m",
+        "de die le",
+        "im",
+        "op ’t",
+        "toe",
+        "Van der",
+        "Zum",
+        "Aus dem",
+        "De l",
+        "In",
+        "Op de",
+        "Tot",
+        "van der",
+        "zum",
+        "aus dem",
+        "de l",
+        "in",
+        "op de",
+        "tot",
+        "Van gen",
+        "Zur",
+        "Aus den",
+        "De l’",
+        "In ’t",
+        "Op den",
+        "Uijt",
+        "van gen",
+        "zur",
+        "aus den",
+        "de l’",
+        "in ’t",
+        "op den",
+        "uijt",
+        "Van het",
+        "Aus der",
+        "De la",
+        "In de",
+        "Op der",
+        "Uijt ’t",
+        "van het",
+        "aus der",
+        "de la",
+        "in de",
+        "op der",
+        "uijt ’t",
+        "Van la",
+    ];
+
+    public function getSuffixes(): array
+    {
+        return self::SUFFIXES;
```

Comment:
Since you are not doing any transformations in here, the mappers would try to match the suffixes by the keys provided in the array above (see https://github.com/theiconic/name-parser/blob/master/src/Mapper/SuffixMapper.php#L42). This means the input would have to contain e.g. `bacc.alaureus` for the suffix to be recognised and it would then be normalised to 'bacc.'. I'm pretty sure that's not what you'd want and a solution like the `array_combine` below, with provision of properly normalised array values could be a solution.